packages/shared/src/workspaces/helpers/features.ts

```diff
@@ -97,63 +76,42 @@ export type WorkspacePlanConfig<Plan extends WorkspacePlans = WorkspacePlans> =
 }
 
 const baseFeatures = [
-  WorkspacePlanFeatures.Workspace,
-  WorkspacePlanFeatures.RoleManagement,
-  WorkspacePlanFeatures.GuestUsers,
-  WorkspacePlanFeatures.PrivateAutomateFunctions
+  WorkspacePlanFeatures.AutomateBeta,
+  WorkspacePlanFeatures.DomainDiscoverability
 ] as const
 
-const teamFeatures = [...baseFeatures]
-
-const proFeatures = [
-  ...teamFeatures,
-  WorkspacePlanFeatures.DomainSecurity,
-  WorkspacePlanFeatures.SSO,
-  WorkspacePlanFeatures.CustomDataRegion,
-  WorkspacePlanFeatures.PrioritySupport
-]
-
 export const WorkspacePaidPlanConfigs: {
   [plan in PaidWorkspacePlans]: WorkspacePlanConfig<plan>
 } = {
   // Old
```

Comment:
If would like to remove these old ones, seems they are not really used on the BE yet, FE doesnt use them anymore, but Im not sure @gjedlicska

Reply:
for now we cannot do it, post release, we can clean up all refs to the old plans